content/en/docs/contribute/style/diagram-guide.md

```diff
@@ -242,8 +207,12 @@ Figure 17. A to B
 more text
 ```
 {{< note >}}
+<<<<<<< HEAD
+You must include the Hugo Mermaid shortcode tags at the start and end of the Mermaid code block. You should add a diagram
+=======
 You must include the Hugo Mermaid shortcode
 tags at the start and end of the Mermaid code block. You should add a diagram
+>>>>>>> 580c643e50ac2aaac0b1b8f8781716a9429c003b
```

Comment:
Merge conflict.

Reply:
Hi @Sea-n, thanks for reviewing. This PR is for experimenting and testing inconsistent Mermaid diagram rendering. I will resolve merge conflict and Netlify deploy. If you can, check some of the pages with Mermaid diagrams (I will post in this PR). Once we see consistent results, I can close this PR and hack up a new one that will be merged and solve the problem.

Tx again.

Copying @sftim